### PR TITLE
TC 00.01.12 | New item > Freestyle Project> Create Freestyle Project

### PR DESCRIPTION
**Pull request by bahasadyr**
https://trello.com/c/PDNFt9wf/539-tc-000112-new-item-freestyle-project-create-freestyle-project

### CODE REVIEW

**Review by cyferkiBY — APPROVED** [review]
LGFM!

File: src/test/java/FreestyleProjectTest.java [hunk 1]
```diff
@@ -1,14 +1,13 @@
 import org.apache.commons.lang3.RandomStringUtils;
-import org.openqa.selenium.Alert;
-import org.openqa.selenium.By;
-import org.openqa.selenium.JavascriptExecutor;
-import org.openqa.selenium.WebElement;
+import org.openqa.selenium.*;
```
Comment (cyferkiBY) [inline src/test/java/FreestyleProjectTest.java:2]: Не уверена хорошая ли идея делать импорт все библиотеки Selenium (*).

Comment (bahasadyr) [reply src/test/java/FreestyleProjectTest.java:2]: this was done not by me, I just wrote my code in this class, so as was told on the lesson, not to have many classes with similar names

File: src/test/java/FreestyleProjectTest.java [hunk 2]
```diff
@@ -280,4 +279,21 @@ public void testCreateBuildNowOnFreestyleProjectPage() {
...
+        getDriver().findElement(LINK_FREESTYLE_PROJECT).click();
+        getDriver().findElement(BUTTON_OK_IN_NEW_ITEM).click();
+        getDriver().findElement(BUTTON_SAVE).click();
+        getDriver().findElement(GO_TO_DASHBOARD_BUTTON).click();
+
+        List<WebElement> lstWithElements = getDriver().findElements(By.xpath("//table[@id='projectstatus']//tbody//tr//td//a"));
+        List<String> lstWithStrings = lstWithElements.stream().map(WebElement::getText).collect(Collectors.toList());
+
+        Assert.assertTrue(lstWithStrings.contains(name));
+
```
[the remark this answers is not in the document]
Comment (bahasadyr) [reply src/test/java/FreestyleProjectTest.java]: done!

**Review by NadiaLidt — APPROVED** [review]

**Pull request merged**